src/dotnet/Fable.Core/Import/Node/Querystring.fs

```diff
@@ -0,0 +1,20 @@
+[<AutoOpen>]
```

Comment:
As now the types are not nested in a `*_types` submodule, maybe we don't need `AutoOpen` here (also in Url module below). This means that all types from all modules will be globally accessible when opening `Fable.Import.Node` which is probably not desirable and may lead to confusion as some types will shadow each other.